graylog2-server/src/main/java/org/graylog/security/certutil/CertRenewalServiceImpl.java

```diff
@@ -98,8 +90,14 @@ Date convertToDateViaInstant(LocalDateTime dateToConvert) {
     }
 
     Date calculateThreshold(String certificateLifetime) {
-        final var lifetime = Duration.parse(certificateLifetime).dividedBy(CERT_RENEWAL_THRESHOLD_PERCENTAGE);
-        var validUntil = clock.now(ZoneId.systemDefault()).plus(lifetime).toLocalDateTime();
+        Duration threshold;
+        PeriodDuration lifetime = PeriodDuration.parse(certificateLifetime);
```

Comment:
I thought about that, but since CertRenewalServiceImpl is deprecated and the logic isn't the same in the two classes, I thought doing this when we refactor that would be a better option.